### ISSUE TITLE
Implementation app features

### ISSUE THREAD

**Issue by Turbovix** [tananaev/traccar-client-android#228]
@tananaev , i want to suggest the escoder feature the app (traccar-client-android) the list of installed apps, and this setting was alternative in the app settings, or that there was need for a password to access the settings or uninstall smartphone app.

**Comment by tananaev** [comment]
Can you please write in English and clearly separate the features that you are requesting.

**Comment by Turbovix** [comment]
It would be possible to add an option for the app to stay hidden from the list of installed apps? or password to change or remove it from your smartphone.

**Comment by tananaev** [comment]
No, it's not possible, but have you looked at the hidden version of the app:

https://www.traccar.org/client/

**Comment by Turbovix** [comment]
The source code available on github is the hidden version?

**Comment by Turbovix** [comment]
Thank you, I understand now!

**Comment by tananaev** [comment]
Source code is for both.
